bedrock/firefox/templates/firefox/family/index-b.html

```diff
@@ -0,0 +1,173 @@
+{% block page_desc %}{{_('Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!')}}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_family_index_b' %}
+{% endblock %}
+
+{% block body_id %}firefox{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% call fxfamilynav('firefox') %}
+    {{ download_firefox(dom_id="sticky-download-desktop", alt_copy=_('Download')) }}
+  {% endcall %}
+{% endblock %}
+
+{% block content %}
+
+<section class="content-section" id="intro">
+  <div class="content">
+    <h2>
+      {{ _('Firefox Web Browser: <br>Fast. Private. Safe.') }}
+    </h2>
+
+    <p>
+    {% trans %}
+      Firefox is faster, more private, and always independent. <br>Available on
+      desktop and mobile.
+    {% endtrans %}
+    </p>
+
+    {{ download_firefox(dom_id="download-firefox") }}
+  </div>
+</section>
+
+<section class="content-section" id="fast">
+  <div class="content">
+    <div class="copy">
+      <h3>{{ _('Browse faster with Firefox') }}</h3>
+
+      <p>
+      {% trans %}
+        Firefox is faster than ever before, thanks to a new multi-process
+        architecture that improves browser speed and page-load time.
+      {% endtrans %}
+
+      {% trans url1=url('mozorg.home'), url2='https://blog.mozilla.org/firefox/tag/quantum' %}
+        <a href="{{ url1 }}">Mozilla</a>, the nonprofit that builds Firefox,
+        invented a new programming language to
+        <a rel="external" href="{{ url2 }}">make Firefox faster</a>.
+      {% endtrans %}
+
+      {% trans %}
+        Our rapid browser update schedule means Firefox loads faster and is more
+        stable every time you use it.
+      {% endtrans %}
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="content-section" id="private-browsing">
+  <div class="content">
+    <div class="copy">
+      <h3>{{ _('Get Firefox for Private Browsing') }}</h3>
+
+      <p>
+      {% trans url1='https://support.mozilla.org/firefox/private-browsing',
+               url2='https://addons.mozilla.org/bundles/privacy',
+               url3='https://addons.mozilla.org/category/adblocking',
+               url4='https://addons.mozilla.org/category/search',
+               url5='https://addons.mozilla.org/category/vpn' %}
+        What does your browser history say about you? Who has access to your browser
+        history? Plenty of browsers offer an incognito mode, but only Firefox offers
+        advanced tracking protection: Just
+        <a href="{{ url1 }}">open a private browsing window</a> and enjoy safe
+        browsing again. Enhance your private browsing experience with dozens of
+        <a href="{{ url2 }}">privacy add-ons</a> — for
+        <a href="{{ url3 }}">ad blocking</a>,
+        <a href="{{ url4 }}">private search</a>, <a href="{{ url5 }}">vpn</a> and
```

Comment:
[Not sure](https://en.wikipedia.org/wiki/Acronym#Case) if this should be vpn or VPN.